packages/pluggable-widgets-tools/configs/rollup.config.js

```diff
@@ -45,6 +45,22 @@ const assetsDirName = "assets";
 const absoluteOutAssetsDir = join(absoluteOutPackageDir, assetsDirName);
 const outAssetsDir = join(outWidgetDir, assetsDirName);
 
+const commonExternalLibs = [
+    // "mendix" and internals under "mendix/"
+    /^mendix($|\/)/,
+
+    // "react"
+    /^react$/,
+
+    // "react/jsx-runtime"
+    /^react\/jsx-runtime$/,
+
+    // "react-dom"
+    /^react-dom$/
+];
+
+const webExternal = [...commonExternalLibs, /^big.js$/];
+
```

Comment:
Again, changes here seem also (see my comment on `rollup.config.native`) cosmetic. Was it intentional?

Reply:
Yes, the same